src/data.js

```diff
@@ -2427,7 +2427,7 @@ module.exports = [
       'Developer',
       'Software Engineer',
       'Full Stack',
-      'nodejs',
+      'node',
```

Comment:
```suggestion
      'Node',
```
that would be a duplicate again